### DISCUSSION TITLE
Header Icon

### DISCUSSION THREAD

**Discussion by we-wake-act21** [eclipse-theia/theia#14025] [Q&A]
Hi,

I am trying to add icon to head tab.

<img width="212" alt="image" src="https://github.com/user-attachments/assets/3fd46fc5-1def-46f4-93f3-5966081e3175">

like the one visible in gitHub. 

Please help!!

**Comment by msujew** [comment]
Hey @we-wake-act21,

you need a preload header to add a favicon to your html. See https://github.com/eclipse-theia/theia/issues/3600#issuecomment-808747788.
